src/Groups/group_characters.jl

```diff
@@ -1264,10 +1293,48 @@ end
 
 class_multiplication_coefficient(tbl::GAPGroupCharacterTable, i::Int, j::Int, k::Int) = class_multiplication_coefficient(ZZRingElem, tbl, i, j, k)
 
+@doc raw"""
+    init_class_fusion(subtbl::GAPGroupCharacterTable,
```

Comment:
Why this name? Is `init` short for `initialize`? What is the rationale behind this name? Does this mean something is also stored in `tbl` and/or `subtbl`?

Reply:
The corresponding function in GAP is called `InitFusion`, I did not try to find a completely independent new name. 
The idea is an "initial approximation" of what is possible, which is cheap to compute; other functions then try to refine the information by excluding possible images.
Would `approximate_class_fusion` sound better?
(And no, nothing gets stored in the character tables.)